### DISCUSSION TITLE
Is this project still active/relevant?

### DISCUSSION THREAD

**Discussion by SpacyRicochet** [openfoodfacts/openfoodfacts-swift#24] [General]
Hello folks, 

I was considering contributing to the project, but I'm wondering if the project is still active or relevant. There's also the [Flutter/Dart mobile app](https://github.com/openfoodfacts/smooth-app/), which says it's for both Android and iOS, which makes it look like it replaces this project.

There's some recent activity, but most of that is work in configuration files, with the last real code contribution made in January 2024, as opposed to the other project, which is seeing some great activity.

Any update you can provide regarding this?

**Comment by teolemon** [comment]
@SpacyRicochet We want to provide a SDK for 3rd party apps which want to leverage Open Food Facts, Open Beauty Facts.
We'd need help adding the latest APIs (like cosmetics, prices, knowledge panels…) so that it becomes more enticing to iOS developpers.

**Reply by SpacyRicochet** [reply]
Ah, now it makes more sense. Thanks for the info!
Quite a bit different from what I imagined, so going to look some more into the docs to see if I find a good opening for me 👍
